justfile

```
@@ -120,10 +120,13 @@ fix: devenv
     $BIN/ruff --fix .
     $BIN/djhtml --tabwidth 2 airlock/
 
-
+# run airlock with django dev server
 run *ARGS: devenv
     $BIN/python manage.py runserver "$@"
 
+# run airlock with gunicorn, like in production
+gunicorn *args: devenv
```

wdyt about having them sit next to each other in the alphabetised `just default` list?

```suggestion
run-gunicorn *args: devenv
```